README.md

```diff
@@ -1,21 +0,0 @@
-# DUST
-
-This repo hosts the contracts for the DUST world.
-
-### Running
-
-```
-pnpm run dev
-```
-
-### Installation
-
-Make sure sure you have:
-
-- [Node.js v18](https://nodejs.org/en/download/package-manager)
```

Comment:
We should still keep the installation and run instructions somewhere. Prob in the same/new README under `packages/world`

Reply:
yes! removed by mistake